src/main/resources/application-dev.properties

```diff
@@ -16,19 +16,36 @@ ccd.data.store.api.url=http://localhost:4452
 tornado.url=https://eu.dws3.docmosis.com/api/render
 # tornado-docker
 #tornado.url=http://localhost:9095/api/render
-tornado.accessKey=${TORNADO_ACCESS_KEY:}
+tornado.accessKey=${TORNADO_ACCESS_KEY:ZDQ0ZWNjYWUtZGE2Yi00OWQ4LThlNDAtYjBhZmY4MDViZWJlOjM0MTM4ODc}
 
 case_document_am.url =http://localhost:4455
 
-uk.gov.notify.api.key=${GOV_NOTIFY_API_KEY:someApiKey}
+uk.gov.notify.api.key=${GOV_NOTIFY_API_KEY:tell_something_else-002d2170-e381-4545-8251-5e87dab724e7-81abc579-544a-489d-8303-31dabc09f439}
 referral.template.id=${REFERRAL_TEMPLATE_ID:12345}
 et1Serving.template.id=${ET1SERVING_TEMPLATE_ID:12345}
 et1Serving.respondent.template.id=${ET1SERVING_RESPONDENT_TEMPLATE_ID:12345}
 rejected.template.id=${REJECTED_TEMPLATE_ID:12345}
 et3Notification.template.id=${ET3_NOTIFICATION_TEMPLATE_ID:1234}
+et3Notification.template.myhmcts.id=${ET3_NOTIFICATION_TEMPLATE_MYHMCTS_ID:test}
+et3Notification.template.citizen.id=${ET3_NOTIFICATION_TEMPLATE_CITIZEN_ID:test}
+tseRespondentResponse.template.id=b1249c5c-9d7c-4e39-bb81-9e95fa058ed3
+tseRespondentAcknowledgementRule92No.template.id=84e64ff7-7c27-42b6-a7fa-ea29dd712223
+tseRespondentAcknowledgementRule92Yes.template.id=580028e7-d9ff-49de-96c2-dc9595572ebb
+
+claimant.tse.template.id=c815d073-ee7e-4a22-b119-e313e4e4023e
+respondent.tse.template.id=91db4e44-ce62-44d2-af41-791341a067a6
+tse.admin.record-a-decision.claimant.template.id=4d0fdc95-8f48-4a67-9713-3c8474044ad2
+tse.admin.record-a-decision.respondent.template.id=9cfc6f59-2c89-4fa3-a8f2-a4bdb3500146
+tse.admin.reply.claimant.template.id=5baee9e9-e8c3-4518-a834-7f3bb336ca64
+tse.admin.reply.respondent.template.id=ac13a3d3-3de6-4574-a52c-cc70e943dcac
```

Comment:
make the keys consistent with the style of other keys

Reply:
All keys have been reworked